pkg/dataServer/dataServer_test.go

```diff
@@ -42,25 +43,26 @@ func TestDataServer(t *testing.T) {
 	proxy, err := db.OpenLocal(logger, cfg, DB)
 	testutil.Ok(t, err)
 
-	ctx := context.Background()
-	account, err := rpc.NewAccount(cfg)
+	accounts, err := rpc.NewAccounts(cfg)
 	testutil.Ok(t, err)
 	contract, err := contracts.NewITellor(client)
 	testutil.Ok(t, err)
-	ds, err := CreateServer(ctx, logger, cfg, proxy, client, contract, &account)
+	// We need to unregister prometheus counter.
```

Comment:
Oh, When running the tests and after the `TestDataServer` test executed we seems to need to unregister  prometheus counters so that `TestDataServerOps` could be passed without an `prometheus counter already registered error`!